app/models/collection.rb

```diff
@@ -160,6 +160,12 @@ def thumbnail_present?
     thumbnail_urls.present?
   end
 
+  def draft?
```

Comment:
@nicktiberi @rschenk pls let me know if you have better names for this method, it is basically to check if the collection has any published work or not.

Reply:
updated in the last commit